doc/release/release_0.18.rst

```diff
@@ -0,0 +1,416 @@
+- Moved our testing from Travis to GitHub Actions (#5074)
+- We now build our wheels on GitHub Actions on the main repo using
+  cibuildwheel. Many thanks to the matplotlib and scikit-learn developers for
+  paving the way for us! (#5080)
+
+Other Pull Requests
+-------------------
+
+- Manage iradon input and output data type (#4298)
+- random walker: Display a warning when the probability is outsite [0,1] for a given tol (#4631)
+- MAINT: remove unused cython file (#4633)
+- Forget legacy data dir (#4662)
+- Setup longdesc markdown and switch to 0.18dev (#4663)
+- Optional pooch dependency (#4666)
+- Adding new default values to functions on doc/examples/segmentation/plot_ncut (#4676)
+- Reintroduced convert with a strong deprecation warning (#4681)
+- In release notes, better describe skimage's relationship to ecosystem (#4689)
+- Perform some todo tasks for 0.18 (#4690)
+- Perform todo tasks for 0.17! (#4691)
+- suppressing warnings from gallery examples (#4692)
+- release notes for 0.17.2 (#4702)
+- Fix gallery example mentioning deprecated argument (#4706)
+- Specify the encoding of files opened in the setup phase (#4713)
+- Remove duplicate fused type definition (#4724)
+- Blacklist cython version 0.29.18 (#4730)
+- Fix CI failures related to conversion of np.floating to dtype (#4731)
+- Fix Ci failures related to array ragged input numpy deprecation (#4735)
+- Unwrap decorators before resolving link to source (sphinx.ext.linkcode) (#4740)
+- Fix plotting error in j-invariant denoising tutorial (#4744)
+- Highlight all source lines with HTML doc "source" links (sphinx.ext.linkcode) (#4746)
+- Turn checklist boxes into bullet points inside the pull request template (#4747)
+- Deprecate (min_distance < 1) and (footprint.size < 2) in peak_local_max (#4753)
+- forbid dask 2.17.0 to fix CI (#4758)
+- try to fix ci which is broken because of pyqt5 last version (#4788)
+- Remove unused variable in j invariant docs (#4792)
+- include all md files in manifest.in (#4793)
+- Remove additional "::" to make plot directive work. (#4798)
+- Use optipng to compress images/thumbnails in our gallery (#4800)
+- Fix runtime warning in blob.py (#4803)
+- Add TODO task for sphinx-gallery>=0.9.0 to remove enforced thumbnail_size (#4804)
+- Change SSIM code example to use real MSE (#4807)
+- Let biomed example load image data with Pooch. (#4809)
+- Tweak threshold_otsu error checking - closes #4811 (#4812)
+- Ensure assert messages from Cython rank filters are informative (#4815)
+- Simplify equivalent_diameter function (#4819)
+- DOC: update subpackage descriptions (#4825)
+- style: be explicit when stacking arrays (#4826)
+- MAINT: import Iterable from collections.abc (Python 3.9 compatibility) (#4834)
+- Silence several warnings in the test suite (#4837)
+- Silence a few RuntimeWarnings in the test suite (#4838)
+- handle color string mapping correctly (#4840)
+- DOC: Autoformat docstrings in io.*.py (#4845)
```

Comment:
I think you should remove the "*" here because it's interpreted as italic formatting for what's below.

Reply:
```suggestion
- DOC: Autoformat docstrings in ``io.*.py`` (#4845)
```